src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  --color-col: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  color: var(--color-col);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+}
+
+.nav__list li {
+  margin-right: 20px;
+}
+
+.nav__list li:last-child {
+  margin-right: 50px;
+}
+
+.nav:hover {
+  color: var(--color-col);
+}
+
+.apple {
+  color: var(--color-col);
+  border-bottom: 2px solid var(--color-col);
+  padding-bottom: 20px;
```

Comment:
just give a vertical padding for all `li`'s, but do it with class selector, not tag

Reply:
![image](https://github.com/mate-academy/layout_moyo-header/assets/102289654/03e24314-23ab-42fc-a49c-496c8228e633)
this should fix a problem with non clickable zone below and above link